src/animation/SetStyle.ts

```diff
@@ -0,0 +1,67 @@
+import { Animation } from "./Animation.js";
```

Comment:
You'll have to export this Animation from `animation/index.ts` for the rest of the library to access it.